tm2/pkg/bft/config/config.go

```diff
@@ -196,21 +196,34 @@ func (cfg *Config) ValidateBasic() error {
 // -----------------------------------------------------------------------------
 // BaseConfig
 
+// Base node data structure:
```

Comment:
nit: this is not aligned, and I'd rather it was as godoc documentation for `BaseConfig.RootDir`